NEWS

```diff
@@ -1,3 +1,7 @@
+4.4.0
+
+Add track() API
```

Comment:
replace with "Add the ability to send event data to split service (.track() api)